tools/mtouch/mtouch.csproj

```diff
@@ -452,8 +461,20 @@
     <Folder Include="external\maccore\" />
     <Folder Include="Xamarin.Linker\" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Errors.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <XlfSourceFormat>Resx</XlfSourceFormat>
+      <XlfOutputItem>EmbeddedResource</XlfOutputItem>
+      <LastGenOutput>Errors.Designer.cs</LastGenOutput>
+      <CustomToolNamespace>Xamarin.Bundler</CustomToolNamespace>
+      <LogicalName>Errors.mtouch.resources</LogicalName>
+    </EmbeddedResource>
+  </ItemGroup>
   <Target Name="AfterBuild">
     <!-- This makes sure that just building the csproj will install the updated mtouch.exe, so that tests get it without having to 'make mtouch' manually -->
     <Exec Command="make mtouch" />
   </Target>
+  <Import Project="..\..\packages\XliffTasks.1.0.0-beta.19606.1\build\XliffTasks.targets" Condition="Exists('..\..\packages\XliffTasks.1.0.0-beta.19606.1\build\XliffTasks.targets')" />
+  <Import Project="..\..\packages\XliffTasks.1.0.0-beta.20059.1\build\XliffTasks.targets" Condition="Exists('..\..\packages\XliffTasks.1.0.0-beta.20059.1\build\XliffTasks.targets')" />
```

Comment:
same here, does not make sense to have both

Reply:
I tried removing one and XliffTasks stopped working. I added XliffTasks using VSMac and it added two, I think it needs both.